guide/payments/introduction.md

```diff
@@ -18,13 +18,13 @@ image: https://bitcoin.design/assets/images/guide/payments/payments-preview.jpg
    layout = "full-width"
 %}
 
-# Payments and transactions
+# Payments
 
-At its core, the Bitcoin protocol enables transactions between people without intermediaries, regardless of where in the world they are.
+At its core, Bitcoin enables payments between people without intermediaries, regardless of where in the world they are.
 
-This chapter is meant to give an overview of bitcoin transactions, including sending, receiving, and transaction privacy, with advice and best practices. After this chapter, you should be able to design payment solutions that are well suited to your product's use case and understand best-practices for its implementation.
+This chapter covers bitcoin payment design considerations when building mobile bitcoin applications. Including what they are, how they work and the best practices for designing these user flows. After this chapter, have a good understanding of how to design payment solutions for your bitcoin mobile applications and understand best-practices for its implementation.
```

Comment:
```suggestion
This chapter covers bitcoin payment design considerations when building mobile bitcoin applications. Including what they are, how they work and the best practices for designing these user flows.
```
I don't think the second sentence adds much. Mostly repeats what's already stated in the first sentence.